### PR TITLE
Csv list

### PR DESCRIPTION
**Pull request by farhaz17**

### CODE REVIEW

File: csvtable.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<?php
+$path = $_GET['path'];
```
Comment (prpigitcse) [inline csvtable.php]: Do not use get or post method, get the file path and id from the database of the current logged in user.

**Comment by prpigitcse** [comment]
Discuss with teammates to get the available functions, for database and functions related issues.

File: csvtable.php [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                        <option value="Archived">Archive</option>
+                        <option value="Rejected">Reject</option>
+                    </select>
+                </div>
+                <div class="col-4">
+                    <input class="btn btn-primary" type="submit" name="submit" value="Submit">
+                </div>
+            </div>
+        </form>
+    </div>
```
Comment (prpigitcse) [inline csvtable.php]: Use ajax to submit the form.

File: csvtable.php (hunk 1 above)
Comment (farhaz17) [reply csvtable.php]: There is a list of different users CSV file to update, not only logged-in users

File: csvtable.php (hunk 2 above)
Comment (farhaz17) [reply csvtable.php]: That I have done should push and make a new pull request?

**Pull request merged**